src/php/class-admin-options.php

```diff
@@ -62,8 +62,10 @@ function lazyload_replace_video($return, $data, $url) {
 	    	return $return;
 		}
 
+		$isAmp = function_exists('amp_is_request') && amp_is_request();
+
```

Comment:
@mjaschen Thank you for the contribution, very much appreciated!

This is maybe a bit nit-picky but we can make the code a little bit more readable by using an early return instead of having `!$isAmp && (! is_feed())` in two conditions:

```php
$isAmp = function_exists('amp_is_request') && amp_is_request();
if ($isAmp || is_feed()) return $return;
```

What do you think?

Either way, I'll get to merging and publishing this change on the upcoming weekend.

Thanks again for contributing!